sdk/tests/conformance2/renderbuffers/framebuffer-test.html

```diff
@@ -191,6 +198,127 @@
   gl.bindFramebuffer(gl.FRAMEBUFFER, null);
   wtu.glErrorShouldBe(gl, gl.NO_ERROR,
             "binding default (null) framebuffer should succeed.");
+
+  // attach/detach a 2d texture to one framebuffer binding point, while no attachment to the other binding point.
+  function attachAndDetachTexture(targetA, targetB) {
+    gl.framebufferTexture2D(targetA, gl.COLOR_ATTACHMENT0, gl.TEXTURE_2D, fbtex, 0);
+    wtu.glErrorShouldBe(gl, gl.NO_ERROR,
+            "attaching a texture to read/draw framebuffer binding point should succeed.");
+    gl.getFramebufferAttachmentParameter(targetA,
+                                       gl.COLOR_ATTACHMENT0,
+                                       gl.FRAMEBUFFER_ATTACHMENT_COLOR_ENCODING);
+    wtu.glErrorShouldBe(gl, gl.NO_ERROR,
+            "calling getFramebufferAttachmentParameter with pname = GL_FRAMEBUFFER_ATTACHMENT_COLOR_ENCODING on read/draw framebuffer should succeed.");
+    gl.getFramebufferAttachmentParameter(targetB,
+                                       gl.COLOR_ATTACHMENT0,
+                                       gl.FRAMEBUFFER_ATTACHMENT_COLOR_ENCODING);
+    wtu.glErrorShouldBe(gl, gl.INVALID_OPERATION,
+            "calling getFramebufferAttachmentParameter with pname = GL_FRAMEBUFFER_ATTACHMENT_COLOR_ENCODING on read/draw framebuffer without any attachment should generate INVALID_OPERATION.");
+    gl.framebufferTexture2D(targetA, gl.COLOR_ATTACHMENT0, gl.TEXTURE_2D, null, 0);
+    wtu.glErrorShouldBe(gl, gl.NO_ERROR,
+            "detaching a texture from read/draw framebuffer should succeed.");
+  }
+
+  var readDrawFB = gl.createFramebuffer();
+  gl.bindFramebuffer(gl.FRAMEBUFFER, readDrawFB);
+  attachAndDetachTexture(gl.READ_FRAMEBUFFER, gl.DRAW_FRAMEBUFFER);
+  attachAndDetachTexture(gl.DRAW_FRAMEBUFFER, gl.READ_FRAMEBUFFER);
```

Comment:
These tests look wrong. The readDrawFB object is being bound to both the read and draw attachment points, so when you modify the object attached to the READ_FRAMEBUFFER attachment point and then query the DRAW_FRAMEBUFFER attachment point, you're talking about the same framebuffer object and the tests above should fail.